tests/features/forums.feature

```diff
@@ -0,0 +1,132 @@
+    #Forum 1
+    Then I go to "/admin/structure/forum_en"
+    And I should see the text "Forums contain forum topics. Use containers to group related forums."
+    Then I click "Add forum"
+    And I should see the text "A forum holds related forum topics."
+    And I fill in "Forum name" with "Forum name test"
+    And I select "General discussion" from "Parent"
+    And I press the "Save" button
+    And I should see the text "Created new forum"
+    And I should see the text "test"
+    And I should see the text "edit forum"
+    #Forum 2
+    Then I click "Add forum"
+    And I should see the text "A forum holds related forum topics."
+    And I fill in "Forum name" with "Forum name test 2"
+    And I select "General discussion" from "Parent"
+    And I press the "Save" button
+    And I should see the text "Created new forum"
+    And I should see the text "test"
+    And I should see the text "edit forum"
+    #Topic on forum 1
+    Then I go to "/forums/forum-name-test_en"
+    Then I click "Add new Forum topic"
+    And I should see the text "Create Forum topic"
+    And I fill in "Subject" with "Subject test"
+    And I press the "Save" button
+    And I should see the text "Subject test"
+    And I should see the text "Forum topic subject test has been created."
+    #Move leaving a shadow copy
+    Then I click "Edit"
+    And I should see the text "Edit Forum topic Subject test"
+    Then I check "Leave shadow copy"
+    And I select "Forum name test 2" from "Forums"
+    And I press the "Save" button
+    And I should see the text "Forum topic subject test has been updated."
+    #Issue https://www.drupal.org/node/1123866
+    #Then I go to "/forums/forum-name-test_en"
+    #And I should see the text "This topic has been moved"
+    #Then I go to "/forums/forum-name-test-2_en"
+    #And I should see the text "Subject test"
+  @api
+  Scenario: Locking and disabling comments. Selecting Closed under Comment settings while editing a forum topic will lock (prevent new comments on) the thread. Selecting Hidden under Comment settings while editing a forum topic will hide all existing comments on the thread, and prevent new ones.
+    Given I go to "/forums/general-discussion_en"
+    Then I click "Add new Forum topic"
+    #New forum topic
+    And I should see the text "Create Forum topic"
+    And I fill in "Subject" with "Subject test"
+    And I press the "Save" button
+    And I should see the text "Subject test"
+    And I should see the text "Forum topic subject test has been created."
+    #New comment
+    And I fill in "Subject" with "Subject comment test"
+    And I fill in "Comment" with "Comment test content"
+    And I press the "Save" button
+    And I should see the text "Your comment has been posted."
+    #Close comments
+    Then I click "Edit"
+    When I select the radio button "Closed Users cannot post comments, but existing comments will be displayed." with the id "edit-comment-1"
```

Comment:
This test is failing on ec_europa. @drupol  can you check with Kevin what should be the label of the "closed comments" feature?